server/api/router/OrderRouter.js

```diff
@@ -1,18 +1,33 @@
 const router = require('express').Router();
-const { Order, User, OrderProduct } = require('../../db/index');
+const { Order, User, OrderProduct, Product } = require('../../db/index');
 
 // API/orders
 router.get('/', async (req, res) => {
   try {
-    const orders = await Order.findAll();
+    const orders = await Order.findAll({include: [{model: Product, through: {attributes: ['productQuantity']}}]});
     res.json(orders);
   } catch (e) {
-    console.log(e => console.error('Could not get Orders from database', e));
-    res.sendStatus(500);
-    //res.redirect('/')
+    console.error(e);
   }
 });
 
+router.post('/', async (req, res) => {
+//   try {
+//     const { id, user, session, orderTotal, product } = req.body;
+//     const order = await Order.create({id, orderTotal});
+//     if (req.userId) {
+//       await Order.update{(user);
+//     }
+//     const orderProducts = await OrderProduct.create();
+//     await order.addProduct
+//     const joinedOrder = Order.findByPk(order.id, {include: [OrderProduct]});
```

Comment:
Don't know why commented out... Will work if you change this line to the one above:

`Order.findByPk(order.id, {include: [{model: Product, through: {attributes: ['productQuantity']}}]})`

Reply:
we did have it set up that way at one point.